src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="field">
+          Surname:
+          <input
+          type="text"
+          id="surname"
+          name="attribute">
```

Comment:
```suggestion
          <input
            type="text"
            id="surname"
            name="attribute"
          >
```
Открывающая и закрывающая скобки на одной линии, атрибуты сдвинуты на два отступа вправо